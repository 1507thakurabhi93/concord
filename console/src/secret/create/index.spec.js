import React from 'react';
import { shallow } from 'enzyme';
import { SecretCreate } from './';

describe('Secret Create Container', () => {
        
    it('should match snapshot', () => {
        const wrapper = shallow(<SecretCreate />);
        expect( wrapper ).toMatchSnapshot();
    })

    it('should render a semantic-ui-react Container', () => {
        const wrapper = shallow(<SecretCreate />);
        expect(wrapper.find('Container').length).toEqual(1);
    });

})