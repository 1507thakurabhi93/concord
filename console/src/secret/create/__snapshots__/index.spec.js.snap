// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Secret Create Container should match snapshot 1`] = `
<Container>
  <SegmentGroup>
    <Segment>
      <ReduxForm />
      <ReduxForm />
      <ReduxForm />
      <ReduxForm />
      <Message
        hidden={true}
        success={true}
      />
      <Message
        error={true}
        hidden={true}
      />
    </Segment>
  </SegmentGroup>
</Container>
`;
